### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by tsalo**
References #110.

Changes proposed:

- Add bug report and feature request templates.
- Add link to NeuroStars for usage questions.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Usage question
+    url: https://neurostars.org/tags/c/software-support/234/babs
```
Comment (zhao-cy) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Hmmm that page does not exist... Maybe `234` is for xcp-d? What I should do to make a page for BABS - create a new question on neurostars?

**Review by zhao-cy — COMMENTED** [review]
Hi @tsalo ! Thank you so much for adding these templates! I have a question here - please see below.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (tsalo) [reply .github/ISSUE_TEMPLATE/config.yml:3]: 234 is for all software-support questions. The problem is that there are no babs-tagged questions yet. I've got some permissions on NeuroStars, so I just created one. The link should work now.

Comment (zhao-cy) [reply .github/ISSUE_TEMPLATE/config.yml:3]: Perfect! Thanks a lot! I'll merge your changes to main after I add to docs that users can ask questions on neurostars!

**Comment by zhao-cy** [comment]
TODO for myself:
- [ ] add to docs that users can also ask questions on neurostars
- [ ] bug report template: 
  - [ ] delete docker version
  - [ ] add DataLad etc versions
  - [ ] ask the users to report printed messages from `babs-check-setup`

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (zhao-cy) [reply .github/ISSUE_TEMPLATE/config.yml:3]: Okay based on Matt's thoughts on 9/15/2023 ("it's a bit early for neurostars") in the BABS slack channel, I think I'll remove the option of neurostars and this `config.yml` file for now.

**Comment by zhao-cy** [comment]
This resolves issue #147

**Pull request merged**